### PR TITLE
Sandbox, Job Title and Skill Endpoints

### PR DESCRIPTION
**Pull request by vvillait88**
## Description of the change

> Added support for our new Sandbox, Job Title Enrichment, and Skill Enrichment endpoints.  Added a note to the readme to help with TS support.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Chore (cleanup or minor QOL tweak that has little to no impact on functionality)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by nebster100 — APPROVED** [review]

**Review by valeriy-demchenko — APPROVED** [review]

**Review by lilahuizenga — APPROVED** [review]

File: example/index.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ PDLJSClient.company.search.elastic({
   console.log(error);
 });
 
-// Supporting APIs
+// // Supporting APIs
```
Comment (vrajur-pdl) [inline example/index.js:76]:
```suggestion
// Supporting APIs
```

File: example/index.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ PDLJSClient.person.retrieve({ id: 'qEnOZ5Oh0poWnQ1luFBfVw_0000' }).then((data) =
   console.log(error);
 });
 
-// Company APIs
+// // Company APIs
```
Comment (vrajur-pdl) [inline example/index.js:53]:
```suggestion
// Company APIs
```

File: README.md [hunk 3]
```diff
@@ -229,6 +299,15 @@ PDLJSClient.school.cleaner({ name: 'university of oregon' }).then((data) => {
...
 | [School Cleaner API](https://docs.peopledatalabs.com/docs/cleaner-apis#schoolclean) | `PDLJS.school.cleaner(...params)` |
+| [Job Title Enrichment API](https://docs.peopledatalabs.com/docs/job-title-enrichment-api) | `PDLJS.jobTitle(...params)` |
+| [Skill Enrichment API](https://docs.peopledatalabs.com/docs/skill-enrichment-api) | `PDLJS.skill(...params)` |
+
+**Sandbox Endpoints**
+| API Endpoint | PDLJS Function |
+|-|-|
+| [Person Enrichment Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | `PDLJS.person.enrichment(...params)` |
+| [Person Search Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | SQL: `PDLJS.person.search.sql(...params)` <br/> Elasticsearch: `PDLJS.person.search.elastic(...params)`|
+| [Person Identify Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | `PDLJS.person.identify(...params)` |
```
Comment (vrajur-pdl) [inline README.md:310]:
```suggestion
| [Person Enrichment Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | `PDLJS.person.enrichment(...params, sandbox: true)` |
| [Person Search Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | SQL: `PDLJS.person.search.sql(...params)` <br/> Elasticsearch: `PDLJS.person.search.elastic(...params, sandbox: true)`|
| [Person Identify Sandbox API](https://docs.peopledatalabs.com/docs/sandbox-apis) | `PDLJS.person.identify(...params, sandbox: true)` |
```

**Review by vrajur-pdl — APPROVED** [review]
Couple suggestions on the README but otherwise looks fine to me

**Pull request merged**